_i18n/en/general/privacy.md

```diff
@@ -1,57 +1,82 @@
+## Online services
+
+### Data from services hosted by the team
+- **The forum**: AntennaPod's forum uses the Discourse software, hosted by the core team. Please see the [forum's privacy policy](https://forum.antennapod.org/privacy) for more details.
+
+### Data from services hosted by other parties
+
+#### The donation system
+AntennaPod uses Open Collective, operated by Open Collective Inc. (USA) and Open Collective Europe ASBL (Belgium). When you make a donation, data flows to:
+- these entities - for more information, see [Open Collective Inc.'s privacy policy](https://opencollective.com/privacypolicy)
+- unless contributing as an [anonymous guest](https://docs.opencollective.com/help/financial-contributors/payments#contributing-as-a-guest) or [incognito](https://docs.opencollective.com/help/financial-contributors/payments#select-a-contributor) user, the AntennaPod team gets access to (but will not extract) your:
   - public profile information (name, company, description, image, Twitter, GitHub, website)
   - email address
 
-## Updates of this Privacy Policy
-The developers may update this policy in the future. It is advisable to check the policy periodically for any changes. Changes to this Privacy Policy are effective when they are made on this document.
+#### The website
+AntennaPod's website is hosted via GitHub Pages.
+- When you visit our website, you are interacting with GitHub, Inc. or GitHub B.V. - see their [privacy policy](https://docs.github.com/en/site-policy/privacy-policies/github-general-privacy-statement) for the data they collect and process.
+- The website does not set any cookies and does not use third-party tracking, analytics or other services. The 'Translate' page in the 'Contribute' section contains an embedded graph which retrieved from the servers of Weblate s.r.o. ([privacy policy](https://weblate.org/en-gb/privacy/)).
+
+#### Google play
+When you actively leave a review in the Google Play store, the AntennaPod team gets access to your public profile information and may extract and store the following in any online service used by the team (e.g. GitHub or the forum):
+- review text
+- AntennaPod version
+- device type
+- device language
+- Android version
+
+#### Weblate
+When you contribute to AntennaPod's translations:
+- you are interacting with the hosted service of Weblate s.r.o. - see their [privacy policy](https://weblate.org/en-gb/privacy/) for the data they collect and process.
+- the AntennaPod team gets access to:
+  - The 'Username', 'Full name' and 'Account e-mail' as set in [your profile](https://hosted.weblate.org/accounts/profile/#account)
+  - When you joined the project and detailed [statistics](https://docs.weblate.org/en/latest/devel/reporting.html#stats) about your contributions, as well as the [languages](https://docs.weblate.org/en/latest/devel/reporting.html#credits) you contribute to and the date and timestamps of your contributions.
```

Comment:
I would leave out the details here and just mention Weblate's privacy policy. If we mention things explicitly here and Weblate changes, we would be stating wrong stuff